controllers/pod-mutator/scheduler/scheduler.go

```diff
@@ -101,9 +65,9 @@ func (p *PodSchedulerSetter) mutatePodsFn(ctx context.Context, pod *corev1.Pod,
 
 	// Skip mutation if the pod annotation has false schedule annotation.
 	if val, exists := pod.ObjectMeta.Annotations[p.SchedulerAnnotationKey]; exists {
-		boolVal, err := strconv.ParseBool(val)
+		enabled, err := strconv.ParseBool(val)
 		// No error in parsing and the value is false, skip the pod.
-		if err == nil && !boolVal {
+		if err == nil && !enabled {
 			log.V(4).Info("pod has storageos.com/scheduler=false annotaion, skipping")
```

Comment:
```suggestion
			log.V(4).Infof("pod has %s=false annotaion, skipping", p.SchedulerAnnotationKey)
```

Does it make sense to use the const you have defined already? I suppose there's not much chance of it changing. Really I'm just curious about whether you chose to not use the const.

Reply:
yeah, it probably does.  With `logr` they recommend against vars in the message and encourage k=v pairs instead, but that doesn't work well here.  Changed.